### PR TITLE
Add Main menu in Zoomx

### PR DESCRIPTION
**Pull request by fathallah92**

### CODE REVIEW

File: zoomx/src/main/java/com/zoomx/zoomx/manager/ZoomX.java [hunk 1]
```diff
@@ -30,4 +32,6 @@ private static void setupDataBase() {
     public static RequestDao getRequestDao() {
         return requestDao;
     }
+
```
Comment (iabdelgawaad) [inline zoomx/src/main/java/com/zoomx/zoomx/manager/ZoomX.java]: remove space

File: zoomx/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1 +1,11 @@
-<manifest package="com.zoomx.zoomx" />
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.zoomx.zoomx">
+
+    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
+
+    <application>
+        <activity android:name=".ui.requestlist.request.RequestActivity" />
+
+        <service android:name=".ui.requestlist.menu_view.ZoomxMenuService" />
```
Comment (iabdelgawaad) [inline zoomx/src/main/AndroidManifest.xml]: What do you think about renaming menu_view to menu as a package name? also try to put it under UI package directly.

File: app/src/main/java/com/zoomx/zoomx/MainActivity.java [hunk 3]
```diff
@@ -78,79 +69,7 @@ public void onComplete() {
...
-            // Show alert dialog to the user saying a separate permission is needed
-            // Launch the settings activity if the user prefers
-            Intent myIntent = new Intent(Settings.ACTION_MANAGE_OVERLAY_PERMISSION);
-            startActivity(myIntent);
-            wm.addView(overLayImage, params);
-        } else {
-            wm.addView(overLayImage, params);
-        }
-    }
+
```
Comment (iabdelgawaad) [inline app/src/main/java/com/zoomx/zoomx/MainActivity.java:72]: remove space

File: zoomx/src/main/AndroidManifest.xml (hunk 2 above)
Comment (fathallah92) [reply zoomx/src/main/AndroidManifest.xml]: I will rename it. You renamed the package to view so I need first to merge with you then refactor it

File: app/src/main/java/com/zoomx/zoomx/MainActivity.java (hunk 3 above)
Comment (fathallah92) [reply app/src/main/java/com/zoomx/zoomx/MainActivity.java:72]: done

File: zoomx/src/main/java/com/zoomx/zoomx/manager/ZoomX.java (hunk 1 above)
Comment (fathallah92) [reply zoomx/src/main/java/com/zoomx/zoomx/manager/ZoomX.java]: done

**Pull request merged**